### ISSUE TITLE
IE11 forEach error - javascript broken :exclamation:

### ISSUE THREAD

**Issue by daveykropf** [kaliberjs/build#122]
Since version 0.53 the Kaliber build doesn't work (no events are being triggered) on IE11 (and maybe older) browsers.

In the console I see the following error (in Dutch):

```java
SCRIPT438: De eigenschap of methode forEach wordt niet ondersteund door dit object
0.b11846281eec2e7cf4db.js (84,1)`
```

When opening the corresponding file, the error is created by the following code: 
```javascript
const elements = document.querySelectorAll('*[data-componentid="App_js"]')
elements.forEach(element => {
  const props = JSON.parse(element.dataset.props)
  hydrate(<Component {...props} />, element)
})
```

To re-produce the problem I made a simple and clean application using the scaffold template and used the following files:
- index.html.js
```javascript
import App from './App?universal'

import stylesheet from '@kaliber/build/lib/stylesheet'
import javascript from '@kaliber/build/lib/javascript'

export default (
  <html lang='nl'>
    <head>
      <meta charSet='utf-8' />
      <title>@kaliber/build</title>
      <meta name='description' content='' />
      <meta name='viewport' content='width=device-width, initial-scale=1' />
      {stylesheet}
      <script defer src='https://cdn.polyfill.io/v2/polyfill.min.js?features=default,es2015,es2016,es2017' />
      {javascript}
    </head>
    <body>
      <App />
    </body>
  </html>
)
```

- App.js
```javascript
export default class App extends Component {
  state = {
    name: ''
  }

  render () {
    const { name } = this.state

    return (
      <form onSubmit={this.onSubmit}>
        <input type='text' name='name' onChange={this.onChange} value={name} />
        <input type='submit' value='submit!' />
      </form>
    )
  }

  onChange = evt => {
    this.setState({ name: evt.currentTarget.value })
  }

  onSubmit = evt => {
    evt.preventDefault()
    alert(`Submit: ${this.state.name}`)
  }
}
```

**Comment by EECOLOR** [comment]
Thank you for the detailed bug report. It seems that the query selector returns something different than we expected.

**Comment by EECOLOR** [comment]
Ahh, no IE support for `forEach` on NodeList: https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach

There is a polyfill: https://github.com/Financial-Times/polyfill-service/pull/1710/files

There was uncertainty about the spec: https://github.com/Financial-Times/polyfill-service/issues/1233

Not sure if it's available or not. We could easily just use a for-loop here. I broke my laptop, so will have to do it tomorrow.

**Comment by klaasman** [comment]
Should be fixed with the installation of `v0.0.57`
